### ISSUE TITLE
Email Notification No Longer "From" the Email Address Submitted in the Form And a Couple of Other Points

### ISSUE THREAD

**Issue by wolfpurplemoon** [tlovett1/custom-contact-forms#27]
I have been using CCF for several years and the latest update broke several features I made use of.

Firstly, and most importantly for me, something that I found very useful was that the emails I received from form submissions had the email address the person had submitted into the form already in the From field. Now the emails all come from Wordpress@(MYWEBSITE).com and I will have to copy the email address from the body of the message in order to reply to people.

The titles of all the fields in the emails I receive now come from the Slug instead of the easy to understand Label 

There is no longer any way to customise the Subject of the email notification

There doesn't seem to be an easy way to customise the look of the form, my old form was styled to fit with the colours of my website and now it is just a bunch of boring grey boxes. In addition - the little help tool tips that used to pop up when people clicked a field to fill it in have gone with no way to add them. And there is no longer a way to customise the text that pops up next to required fields when they are missed (It says "Please Fill Out This Field" which is too American for my tastes!)

The slightly more spaced out way of displaying the form in an email is good in some ways - but it just doesn't look as nice as the old one, it'd be good to be able to customise the formatting of the email - especially as clients will see how it looks to me when I reply to them.

**Comment by Bugzippy** [comment]
I see the form is received from wordpress to only the first email but even when inserting a comma between emails only the first submission email is received but NOT THE SECOND EMAIL

**Comment by tlovett1** [comment]
Setting the from email will be re-added in a later version.

Yes, slugs replaced label. Perhaps we can change back to label.

I just pushed a filter to make subject more easily customizable.

Forms now inherit the styling from your theme which I think is cleaner and better in the long run.

I just pushed a filter to make the email message more easily customizable.

You can customize form error messages via a filter just pushed.

**Comment by tlovett1** [comment]
@Bugzippy I receive both emails. Anymore details?

**Comment by wolfpurplemoon** [comment]
Thanks for your reply - I don't see these new options on my dashboard yet - were they included in today's plugin update?

Regarding slugs replacing labels, labels would be far more useful as then I can see the title of each field as the client sees them rather than slightly nonsensical 'database friendly' versions of them.

Thanks again for your quick response!

**Comment by wolfpurplemoon** [comment]
I just applied the latest update and I can now see and set the option for the email notification to be from the email address in the Email field. However, though I have set it the From field then had 'null' as a value when I test the form.

Also, when submitting the form it was just bringing up the little spinning loading circle for a second or two and then the form would stay as it is rather than showing text but although the form was submitting and I received my email notifications it is no longer obvious that it has gone which could lead to multiple submissions.

In testing some possible causes, I first tried removing the "Require Confirmation" option in the Email field and now both problems are fixed, so this seems to be the culprit!
